src/auth/auth.action.js

```diff
@@ -26,6 +33,19 @@ export const auth = (code, state) => {
   };
 };
 
+export const signOut = callback => {
```

Comment:
Maybe we can return a promise in here and call the next action later inside the component or something like that.
What do you think @housseindjirdeh